packages/bpk-component-navigation-bar/src/bpk-navigation-bar.scss

```diff
@@ -0,0 +1,57 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-navigation-bar {
+  position: relative;
```

Comment:
We should be using flexbox here - it is preferred over absolute positioning - you wouldn't have to handle RTL for instance...

Reply:
I don't think there's a way to implement this with flex because both items can present or not, they can have different sizes and the title need to be always centralised. see:

![desktop hd](https://user-images.githubusercontent.com/1457263/39366498-f9374b60-4a2b-11e8-80ef-fadb1872338f.png)

Reply:
Yeah you're right i'd keep as is.